File: blimp/chunks/lang/bench/btree.c
#include <stdio.h>
long long tree_build(long long depth) {
    if (depth == 0) return 1;
    long long left = tree_build(depth - 1);
    long long right = tree_build(depth - 1);
    return left + right + 1;
}
int main() {
    printf("%lld\n", tree_build(25));
    return 0;
}

File: blimp/chunks/lang/bench/btree.py
import sys
sys.setrecursionlimit(3000000)
def tree_build(depth):
    if depth == 0: return 1
    left = tree_build(depth - 1)
    right = tree_build(depth - 1)
    return left + right + 1
print(tree_build(25))
